content/influxdb/cloud/sign-up.md

```diff
@@ -0,0 +1,297 @@
+
+For details, see [Automatically configure Telegraf](/influxdb/cloud/write-data/no-code/use-telegraf/auto-config/)
+and [Manually update Telegraf configurations](/influxdb/cloud/write-data/no-code/use-telegraf/manual-config/).
+
+#### API, CLI, and client libraries
+
+For information about using the InfluxDB v2 API, `influx` CLI, and client libraries to write data,
+see [Write data to InfluxDB](/influxdb/cloud/write-data/).
+
+#### Demo data
+
+**{{< cloud-name "short" >}}** lets you [add a demo data bucket](/influxdb/cloud/reference/sample-data/#influxdb-cloud-demo-data)
+for quick, **free** access to time series data.
+
+### Query data
+
+Query data using Flux, the UI, and the `influx` command line interface.
+See [Query data](/influxdb/cloud/query-data/).
+
+### Process data
+
+Use InfluxDB tasks to process and downsample data. See [Process data](/influxdb/cloud/process-data/).
+
+### Visualize data
+
+Build custom dashboards to visualize your data.
+See [Visualize data](/influxdb/cloud/visualize-data/).
+
+### Monitor and alert
+
+Monitor your data and sends alerts based on specified logic.
+See [Monitor and alert](/influxdb/cloud/monitor-alert/).
+
+## Differences between InfluxDB Cloud and InfluxDB OSS
+
+{{< cloud-name >}} is API-compatible and functionally compatible with InfluxDB OSS 2.0.
+The primary differences between InfluxDB OSS 2.0 and InfluxDB Cloud are:
+
+- [InfluxDB scrapers](/influxdb/v2.0/write-data/no-code/scrape-data/) that collect data from specified
+  targets are not available in {{< cloud-name "short" >}}.
+- {{< cloud-name "short" >}} instances are currently limited to a single organization.
+
+#### New features in InfluxDB Cloud
+
+- **Free Plan (rate-limited)**: Skip downloading and installing InfluxDB OSS 2.0 and
+  jump into exploring InfluxDB Cloud technology.
+  The Free Plan is designed for getting started with InfluxDB and for small hobby projects.
+- **Flux support**: [Flux](/influxdb/cloud/query-data/get-started/) is a standalone data
+  scripting and query language that increases productivity and code reuse.
+  It is the primary language for working with data within InfluxDB Cloud.
+  Flux can be used with other data sources as well, letting you work with data where it resides.
+- **Unified API**: Everything in InfluxDB (ingest, query, storage, and visualization)
+  is now accessible using a unified [InfluxDB v2 API](/influxdb/cloud/reference/api/) that
+  enables seamless movement between open source and cloud.
+- **Integrated visualization and dashboards**: Based on the pioneering Chronograf project,
+  the new user interface (InfluxDB UI) offers quick and effortless onboarding,
+  richer user experiences, and significantly quicker results.
+- **Usage-based pricing**: The [Usage-based Plan](/influxdb/cloud/account-management/pricing-plans/#usage-based-plan)
+  offers more flexibility and ensures that you only pay for what you use.
```

Comment:
I think we could probably remove this content and put it into its own doc. This has changed and will continue to change. We need to be better about updating it.

Reply:
Moving forward, we should prob just mention "new" features in Release Notes. As for this content, we could drop free plan/usage-based pricing given we discuss earlier, and mention Flux support, unified API, and dashboards in intro paragraph of this doc...or leave that content to the get started doc, either or.